### PR TITLE
Flekschas/export overlays as svg

### PR DESCRIPTION
**Pull request by flekschas**
## Description

> What was changed in this pull request?

Implement SVG export of overlays.

Example screenshot from http://localhost:8080/apis/svg.html?/viewconfs/overlay-tracks.json:

Screen Shot 2020-10-23 at 10 03 23 AM

> Why is it necessary?

To see overlays in the exported SVG

## Checklist

- [x] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [ ] Unit tests added or updated
- [ ] Documentation added or updated
- [x] Example(s) added or updated
- [ ] Update schema.json if there are changes to the viewconf JSON structure format
- [x] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/OverlayTrack.js [hunk 1]
```diff
@@ -299,12 +389,13 @@ class OverlayTrack extends PixiTrack {
...
               stroke,
               outline,
+              options,
             ),
           );
         }
 
         if (op.orientation === '1d-vertical' || op.orientation === '2d') {
-          this.options.extent.forEach((extent) =>
+          this.options.extent.forEach(extent =>
```
Comment (pkerpedjiev) [inline app/scripts/OverlayTrack.js:398]: Which version of prettier are you using? The default (in v2) should be to add parens around anonymous function arguments.

File: app/scripts/OverlayTrack.js [hunk 2]
```diff
@@ -234,6 +322,8 @@ class OverlayTrack extends PixiTrack {
       height = minHeight;
     }
 
+    options.isVertical = true;
```
Comment (pkerpedjiev) [inline app/scripts/OverlayTrack.js:325]: Not sure why this is here but I presume it's intentional?

**Review by pkerpedjiev — COMMENTED** [review]
The only reason I'm not approving is the prettier thing. Should be easy to fix.

File: app/scripts/OverlayTrack.js (hunk 2 above)
Comment (flekschas) [reply app/scripts/OverlayTrack.js:325]: Yes it is. it's part of the function to draw a vertical annotation

File: app/scripts/OverlayTrack.js (hunk 1 above)
Comment (flekschas) [reply app/scripts/OverlayTrack.js:398]: Forgot to do `npm i`. Thanks for the catch

**Review by pkerpedjiev — APPROVED** [review]

**Pull request merged**